### PR TITLE
Added tasks four chapter

### PR DESCRIPTION
**Pull request by Shillerm**

### CODE REVIEW

File: cycle_for/task_02.py
Comment (Ken4scholars) [inline cycle_for/task_02.py]: You can do it this way to make it shorter:
```
step = 1 if A < B else -1
for number in range(A, B + step, step):
    print(number)
```

File: cycle_for/task_03.py
Comment (Ken4scholars) [inline cycle_for/task_03.py]: The instruction says you can solve it without `if`. Try that

File: cycle_for/task_04.py
Comment (Ken4scholars) [inline cycle_for/task_04.py]: As the task says - you should use the minimal number of variables. You can still reduce it further like this:
```
sum = 0
for i in range(10):
    sum += int(input())
print(sum)
```

File: cycle_for/task_05.py
Comment (Ken4scholars) [inline cycle_for/task_05.py]: Same as last question. You're using way too many variables here. At every iteration, you're creating a new variable `number`

File: cycle_for/task_06.py
Comment (Ken4scholars) [inline cycle_for/task_06.py]: I don't think the solution is correct. Does it pass all the tests?

**Review by Ken4scholars — CHANGES_REQUESTED** [review]
These are called loops in English and not circles. In Russian, they called цикл. I also made that mistake when I started. Good so far, just a little more attention to the task requirements is needed. 
Try to answer to my comments and refactor the code where needed. Don't merge it until I approve.

File: cycle_for/task_02.py
Comment (Shillerm) [reply cycle_for/task_02.py]: Okey

File: cycle_for/task_03.py
Comment (Shillerm) [reply cycle_for/task_03.py]: a = int(input())
b = int(input())
for i in range(a - (a + 1) % 2, b - b % 2, -2):
    print(i, end=' ')

Comment (Shillerm) [reply cycle_for/task_03.py]:
```
a = int(input())
b = int(input())
for i in range(a - (a + 1) % 2, b - b % 2, -2):
    print(i, end=' ')
```

File: cycle_for/task_06.py
Comment (Shillerm) [reply cycle_for/task_06.py]: What's wrong?

File: cycle_for/task_05.py
Comment (Shillerm) [reply cycle_for/task_05.py]:
```
N = int(input())
sum = 0
for number in range(N):
    sum += int(input())
print(sum)
```

File: cycle_for/task_06.py
Comment (Ken4scholars) [reply cycle_for/task_06.py]: @Shillerm I see now. From the comments, I thought you were supposed to calculate 13 + 23 + ... n3 but it's actually the cube - 1^3 + 2 ^3 +...n^3. I checked the question in the site so now I know it is correct

**Review by Ken4scholars — APPROVED** [review]

**Pull request merged**
